configure.ac

```diff
@@ -640,6 +640,12 @@ AC_CHECK_FUNCS(explicit_bzero memset_explicit)
 AC_CHECK_FUNCS([ruserok_af ruserok], [break])
 AC_CHECK_FUNCS(close_range)
 
+dnl For module/pam_timestamp
+AC_CHECK_HEADERS([sys/random.h unistd.h])
+AC_CHECK_LIB([bsd], [getrandom])
+AC_CHECK_LIB([System], [getrandom])
+AC_CHECK_FUNCS(getrandom)
```

Comment:
If anything, `AC_SEARCH_LIBS` would have to be used instead of `AC_CHECK_LIB`, but it seems to be an overkill in this case.  Let's just use `AC_CHECK_FUNCS(getrandom)`.

Reply:
Updated it to `AC_SEARCH_LIBS` only if `AC_CHECK_FUNCS` succeeds.

That way OpenBSD and OSX should work properly without unnecessary checks for most other platforms.

Reply:
It's the other way around, `AC_SEARCH_LIBS` is called first to find a library that provides the function, see e.g. the dance around `crypt_r` in `configure.ac`.